README.md

```diff
@@ -10,6 +10,9 @@ library of state-of-the-art machine learning algorithms for catalysis.
     <img src="https://user-images.githubusercontent.com/1156489/170388229-642c6619-dece-4c88-85ef-b46f4d5f1031.gif">
 </div>
 
```

Comment:
Can we add the Github Actions badge here as well and remove CircleCI

Reply:
I'll do that in the `ocp_package` branch